README.md

```diff
@@ -1,7 +1,8 @@
 ## wgpu-rs
 [![Build Status](https://travis-ci.org/gfx-rs/wgpu-rs.svg)](https://travis-ci.org/gfx-rs/wgpu-rs)
 [![Crates.io](https://img.shields.io/crates/v/wgpu.svg)](https://crates.io/crates/wgpu)
-[![Gitter](https://badges.gitter.im/gfx-rs/webgpu.svg)](https://gitter.im/gfx-rs/webgpu)
+[![Docs.rs](https://docs.rs/wgpu/badge.svg)](https://docs.rs/wgpu)
+[![Matrix](https://img.shields.io/matrix/wgpu:matrix.org)](https://riot.im/app/#/room/#wgpu:matrix.org)
```

Comment:
Maybe we could create the Matrix link through matrix.to, which should be https://matrix.to/#/#wgpu:matrix.org (mentioned [in the Matrix FAQ](https://matrix.org/faq/#how-do-i-create-a-link-to-a-matrix-room-or-user%3F).

Or we could link to the gfx room instead (https://matrix.to/#/#gfx:matrix.org) – I think @kvark was hoping to use the gfx room primarily, at least initially

Reply:
It looks like `wgpu` room is taking off, regardless of my original intent (to just reserve the name).